src/content/topics/integrations/sleuth.mdx

```diff
@@ -0,0 +1,94 @@
+     </CalloutDescription>
+   </Callout>
+
+## Setting up the Sleuth integration
+
+To make the Sleuth integration with LaunchDarkly:
+
+1. Log into your [Sleuth](https://app.sleuth.io/accounts/login/) Dashboard.
+
+2. In the left sidebar, click the **Create** dropdown, then click **Add feature flags**.
+
+3. Click on the **Enable LaunchDarkly feature flags**. 
+
+4. Click **Authorize** to allow Sleuth to read and modify your LaunchDarkly data. 
+
+<Callout intent="warning">
+  <CalloutTitle>Authorizing Sleuth to access your data</CalloutTitle>
+     <CalloutDescription>
+       Clicking **Deny** prevents the integration from being instantiated. This effectively severs the ability of Sleuth analyzing feature flags as a change source. 
+     </CalloutDescription>
+   </Callout>
+
+![LaunchDarkly authorization dialog](../images/sleuth-ld-integration-auth-dialog.png)
+
+5. The integration is successful when the green **LaunchDarkly feature flags enabled** button is displayed in the Sleuth integration card for Feature Flags. 
+
+![A successful integration!](../images/sleuth-ld-integration-success-dialog_3.png)
+
+## Configuring the Sleuth integration
+
+Now that the Sleuth LaunchDarkly integration has been set up, you must now configure Sleuth to know which feature flags it should track. 
+
+To select a LaunchDarkly project and environment to track:  
+
+1. Click the **LaunchDarkly feature flags enabled** button (shown above). 
+
+2. Select the **Feature Flag Project** and **Environment** you would like to track. The projects and environments shown in the dropdowns are taken from your LaunchDarkly configuration.  
+
+![Feature flag change source setup in Sleuth](../images/sleuth-ld-integration-config-dialog.png)  
+
+![Projects and environments in LaunchDarkly](../images/sleuth-ld-integration-projects-dialog.png)
+
+3. Give the integration a **Name**, then press **Create**. 
+
+4. Once the connection is established, you will be taken to the Sleuth trend graph, where you will be able to monitor how your feature flags impact your code over time. You're set&mdash;now sit back and let Sleuth track your code over time. 
+
+![Sleuth trend graph](../images/sleuth-ld-integration-trendgraph.png)
+
+<Callout intent="info">
+  <CalloutTitle>Tracking multiple projects/environments</CalloutTitle>
+    <CalloutDescription>
+       To track multiple LaunchDarkly projects/environments in Sleuth, create a new Sleuth project. You can create as many Sleuth projects as needed to track any number of feature flag project and environment combinations. 
+       
+       For more information, read <a href="https://help.sleuth.io/projects#creating-a-project">Creating a project</a> in Sleuth Documentation.  
+  </CalloutDescription>
+</Callout>
+
+## Disabling the Sleuth integration
```

Comment:
Would it be more accurate here to say "Removing" or "Deleting" the Sleuth integration? "Disabling" implies to me that the integration remains connected to your LaunchDarkly project but no longer does anything.

Reply:
@scribblingfox The UI on our integrations cards toggle between **connect** and **disconnect**. The LaunchDarkly card, as with all other cards, remains visible even when disconnected (in which case the integration status would read **connect**). I went with "Disabling" because the card never goes away. **Let's change it to "Removing**": It implies the connection is severed but solves for a backend disconnect while allowing for the card to remain visible in the integrations section.